apps_download/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2017 BizzAppDev - Ruchir Shukla <ruchir@bizzappdev.com>
```

Comment:
How is this copyright in the past?

Also, the copyright is typically held by the company and not the individual persons - this prevents numerous duplicate copyrights when someone from the same company works on the file. That is, unless you have a specific agreement with your company in which you are retaining the copyright. In that case, that should be indicated.